lib/matplotlib/axes/_axes.py

```diff
@@ -114,7 +114,8 @@ def get_title(self, loc="center"):
         title = cbook._check_getitem(titles, loc=loc.lower())
         return title.get_text()
 
-    def set_title(self, label, fontdict=None, loc=None, pad=None, **kwargs):
+    def set_title(self, label, fontdict=None, loc=None, y=None, pad=None,
```

Comment:
Can we please make `y` a keyword only?  If not it needs to go after `pad`.

Reply:
made the signature: `def set_title(self, label, fontdict=None, loc=None, pad=None, *, y=None, **kwargs):`